tests/apollo/test_skvbc_backup_restore.py

```diff
@@ -183,7 +183,112 @@ async def test_checkpoint_propagation_after_restarting_all_replicas_under_load(s
                                                                      assert_state_transfer_not_started=False)
 
         next_checkpoint_after = await bft_network.wait_for_checkpoint(replica_id=current_primary)
-        self.assertGreaterEqual(next_checkpoint_before + 1, next_checkpoint_after)
+        self.assertGreaterEqual(next_checkpoint_after, next_checkpoint_before + 1)
+
+    @with_trio
```

Comment:
Given this is an advanced scenario, the focus should be on running it locally and manually reviewing the logs, to make sure everything happens in the right order. I'd suggest to not enable it in CI, as it takes a long time to execute, and also it may be unstable. To do so, you can add the following decorator on top of the test method:
```python
@unittest.skip("Advanced manual scenario - not part of CI")
```

Reply:
done.